src/main/java/nextstep/ladder/view/InputView.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.ladder.view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+import static java.lang.System.in;
+
+public class InputView {
+    private static final Scanner scanner = new Scanner(in);
```

Comment:
`static final` 키워드를 통해 상수로 선언하셨으니, 컨벤션에 따라 이름을 대문자로 지어보면 어떨까요?
```suggestion
    private static final Scanner SCANNER = new Scanner(in);
```